DevAcademy/Services/PlacesService/ProductionPlacesService.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  ProductionPlacesService.swift
+//  DevAcademy
+//
+//  Created by Tomáš Melichar on 16.08.2023.
+//
+
+import Foundation
+
+private struct ApiConstants {
+    // [Documentation] https://data.brno.cz/datasets/5f5ef5229c204723844d075fde3fbd7c_0/api
+    static let url = "https://gis.brno.cz/ags1/rest/services/OMI/omi_ok_kulturni_instituce/FeatureServer/0/query?where=1%3D1&outFields=*&outSR=4326&f=json"
+    /// Current is `Get`
+    static let httpMethod = "GET"
```

Comment:
Ten struct s konstantama super 👍 ...mozna jenom tu metodu GET bych si do nej nepridaval. Kdyz bys potom pridaval nejake dalsi endpointy, mohlo by se ti stat, ze budes potrebovat i jine http metody, tak bych si mozna spis ty konstanty tech metod schoval do nejakeho enumu. Napr.
```Swift
enum HTTPMetod: String {
case get = "GET"
case post = "POST"
case patch = "PATCH"
...
}
```

Reply:
Super tip, upravím to v dalším PR :)